indico/modules/auth/controllers.py

```diff
@@ -54,6 +57,11 @@ def _get_provider(name, external):
     return provider
 
 
+def _sort_providers(providers: list[AuthProvider]) -> list[AuthProvider]:
+    """Sort providers by order and title."""
+    return sorted(providers, key=lambda p: (p.settings.get('order', math.inf), p.title))
```

Comment:
`display_order` might be a better name?
```suggestion
    return sorted(providers, key=lambda p: (p.settings.get('display_order', math.inf), p.title))
```